sos/report/plugins/ovn_central.py

```diff
@@ -125,10 +124,12 @@ def setup(self):
               format(self.ovn_sbdb_sock_path),
               "output": "Leader: self"}
         if self.test_predicate(self, pred=SoSPredicate(self, cmd_outputs=co)):
-            sb_tables = self.get_tables_from_schema(self.path_join(
-                schema_dir, 'ovn-sb.ovsschema'), ['Logical_Flow'])
-            self.add_database_output(sb_tables, sbctl_cmds, 'ovn-sbctl')
-            cmds += sbctl_cmds
+            # backward compatibility
+            for path in ['/usr/share/openvswitch', '/usr/share/ovn']:
+                sb_tables = self.get_tables_from_schema(self.path_join(
+                    path, 'ovn-sb.ovsschema'), ['Logical_Flow'])
+                self.add_database_output(sb_tables, sbctl_cmds, 'ovn-sbctl')
+                cmds += sbctl_cmds
```

Comment:
I believe this should be outside the `for` loop, as otherwise we'd end up adding `sbctl_cmds` twice.

Reply:
You are right! just fixed! thx